node-process-argv/echo.js

```diff
@@ -0,0 +1 @@
+console.log(process.argv.slice(2).join(' '));
```

Comment:
Nice! But `console.log(process.arv[2])` would have been enough

Reply:
ah my bad, I didn't see that the example was putting a single string argument.